consent/strategy_default.go

```diff
@@ -41,6 +41,7 @@ import (
 )
 
 const (
+	DeviceVerificationPath      = "/oauth2/device/verify"
```

Comment:
Is this URL coming from some spec?

Reply:
No, the spec does not specify how the user_code should be verified.

Reply:
thanks! i think it's pretty consistent with existing api naming for hydra so LGTM :)